common-crafting.lic

```diff
@@ -229,8 +229,8 @@ module DRCC
         order_enchant(stock_room, stock_needed, stock_number, bag, belt)
       end
     when /I could not find what you were referring to./
-      case bput('tap my fount on my brazier', 'You tap .* atop a brazier.', 'I could not find what you were referring to.')
-      when /You tap (.*) atop a brazier./
+      case bput('tap my fount on my brazier', /You tap .* atop a .* brazier./, /I could not find what you were referring to./)
```

Comment:
Do you actually want `You tap .* atop a .*brazier` so that it matches both:
- You tap a burin atop a fancy brazier
- You tap a burin atop a brazier
I don't enchant, but if this was working for some people before, with the second string, it looks like this would break them.

Reply:
This concerns me too.

`atop a .* brazier` won't match `atop a brazier` because of the space

Reply:
The regex tester worked with a space or no space. :man_shrugging: I removed the space.